tools/retry

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bash
```

Comment:
Maybe just do a quick version of this in python

Reply:
What do you think of using https://github.com/strizhechenko/shell-retry?